tests/ttnn/unit_tests/operations/test_matmul.py

```diff
@@ -1229,14 +1229,14 @@ def test_matmul_with_matched_width_height(device, m_size, k_size, n_size):
 def test_matmul_with_matched_width_height_from_1D(device, k_size, n_size):
     torch.manual_seed(0)
 
-    torch_input_tensor_a = torch.rand((k_size), dtype=torch.bfloat16)
+    torch_input_tensor_a = torch.rand((1, k_size), dtype=torch.bfloat16)
```

Comment:
`(k_size)` wouldn't work anymore?

Reply:
Yes. The handling of one dimensional tensors changed to be actually correct, so the shape is now preserved (just `(ksize)`), but before it was padded to 2d (like `(1, ksize)`)